contracts/integrations/test/bridges/curve_bridge_mainnet_test.ts

```diff
@@ -0,0 +1,57 @@
+import { artifacts as assetProxyArtifacts } from '@0x/contracts-asset-proxy';
+import { CurveBridgeContract } from '@0x/contracts-asset-proxy/lib/src/wrappers';
+import { ERC20TokenContract } from '@0x/contracts-erc20';
+import { blockchainTests, constants, describe, toBaseUnitAmount } from '@0x/contracts-test-utils';
+import { AbiEncoder } from '@0x/utils';
+
+blockchainTests.fork.skip('Mainnet curve bridge tests', env => {
```

Comment:
I think by default we now run all fork tests, so can probably use `fork.resets`.